app/controllers/HashtagContentController.java

```diff
@@ -15,6 +16,7 @@
  */
 public class HashtagContentController extends Controller {
 
+    private TestDataAccess testDataSource = new TestDataAccess();
```

Comment:
the "new TestDataAccess" should be replacing the "RedisAccessObject", not having both. Also, unless this is somehow invalidated by the way play does things, this should be set up in a way that allows us to give the controller, or if necessary an intermediary, the type of AbstractDataAccess to use, then we don't need to physically change the class between testing and the real thing. 

Might want to check if there is a "proper" way to do that in play but typical options are a setter method or a configuration file (so you can have a running config and a testing config)